src/server/graphql/mutations/createTask.js

```diff
@@ -43,7 +43,7 @@ export default {
 
     // RESOLUTION
     const teamMemberId = toTeamMemberId(teamId, userId);
-    const taskId = `${teamId}::${shortid.generate()}`;
+    const taskId = shortid.generate();
```

Comment:
Will this require a migration?

Reply:
nah because now the ID is treated like an opaque string by everything. if that opaque string happens to have some characters in it that align with an existing teamId then it doesn't matter.